qt/io.github.bit_team.back_in_time.gui.metainfo.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Docu:
+        https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html
+
+    Validate (lint) on shell:
+        appstreamcli validate *xml
+-->
+
+
+<component type="desktop-application">
+  <id>io.github.bit_team.back_in_time.gui</id>
+  <name>Back In Time</name>
+  <summary>Backup tool for GNU Linux desktop using rsync with its hard-links feature to save storage space</summary>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0</project_license>
+
+  <description>
+    <p>
+      It is an easy-to-use backup tool for files and folders. It runs on GNU
+      Linux and provides a command line tool "backintime" and a Qt5 GUI
+      "backintime-qt". It uses "rsync" in the back to take manual or scheduled
+      snapshots and stores them locally or remotely through SSH. Each snapshot
+      is its own folder with copies of the original files, but unchanged files
```

Comment:
...is **in** its own folder...?

Reply:
Both sounds OK to me. But my English shouldn't be the reference. 😆 
Modified it, also in REAMDE.md.